core/src/main/java/com/scalar/db/api/SelectionInterface.java

```diff
@@ -0,0 +1,14 @@
+package com.scalar.db.api;
+
+import java.util.List;
+
+/** An interface for selection operations. */
+public interface SelectionInterface extends BaseOperationInterface {
+
+  /**
+   * Returns the table name for this operation
```

Comment:
"column names"?

Reply:
Good catch! Thank you! Fixed in d6da904.